README.md

```diff
@@ -4,6 +4,8 @@ Adds fields to the tender object to describe the legal basis of the procedure.
 
 The `tender.legalBasis` field is a `Classification` object. Example classification schemes are [LEX](https://en.wikipedia.org/wiki/Lex_(URN)), [CELEX](https://eur-lex.europa.eu/content/help/faq/intro.html#help8) and [ELI](https://en.wikipedia.org/wiki/European_Legislation_Identifier).
 
+For the specific case of disclosing whether tenders are open to international suppliers under a treaty, use the [coveredBy extension](https://extensions.open-contracting.org/en/extensions/coveredBy/master/).
```

Comment:
We can omit the `master/`, so that if we create a versioned release, it will redirect to it instead of master.